libs/caseStatuses.js

```diff
@@ -11,9 +11,9 @@ const statuses = [
       'Du har påbörjat en ansökan. Du kan öppna din ansökan och fortsätta där du slutade.',
   },
   {
-    type: 'active:signed:viva',
+    type: 'active:signature:completed',
```

Comment:
I guess it somehow follows the status in terms of hierarchy of information. But i can see the confusion. My thought was to categories the signature status into a group that then have two states completed (active:signature:completed) and pending (active:signature:pending).

Mostly because it felt weird to add a status with camelcase like active:pendingSignature. But i'm open for suggestion :)

Reply:
Agree that camel case feels a bit weird in this context, however we already have several camel case statuses such as `partiallyApproved`, `completionRequired` etc. so for consistency I prefer that approach.

Grouping by `signature` also seems like a good idea however, so perhaps this design discussion needs more input.

Reply:
I can change it to active:signed, for these changes then we can tackle this issue when we implement the pending/completed flow for signatures